docs/Analyzer Configuration.md

```diff
@@ -1,10 +1,12 @@
+<!-- Markdown used in this file is tweaked for GitHub rendering, it's possible that it renders differently in other tools (e.g. VS Code) -->
```

Comment:
If that happens, then we're violating some markdown rules.
It *should* render the same.

Reply:
Nope, each rendering engine is doing a different "interpretation" of markdown.

Reply:
That's why you have such extensions: https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles

Reply:
I think there's always a way to tweak things to render correctly in both.

Reply:
For example, see the rationale written for MD022

https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

> Rationale: Aside from aesthetic reasons, some parsers, including kramdown, will not parse headings that don't have a blank line before, and will parse them as regular text.

Reply:
If you find a way to have things to render correctly please I am open to suggestion :)

Reply:
I haven't tried, but the suggestion in https://github.com/dotnet/roslyn-analyzers/pull/4541#discussion_r539356934 may work.

IIRC, I've seen this in dotnet/docs

Reply:
MD0033 no inline html code :)

Reply:
It's a choice between having consistency over different parsers vs. violating MD033. Probably we should have a feature request for markdownlint to detect such cases and exclude them for MD033 :smile: